catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrProvider.java

```diff
@@ -1840,7 +1840,7 @@ public void testQueryMissingSortField() throws IngestException, UnsupportedQuery
 
         QueryImpl query = new QueryImpl(txtFilter);
 
-        query.setSortBy(new ddf.catalog.filter.SortByImpl("unknownField", SortOrder.ASCENDING));
+        query.setSortBy(new ddf.catalog.filter.impl.SortByImpl("unknownField", SortOrder.ASCENDING));
```

Comment:
Ah, I take it back, yeah it is still needed because org.geotools.filter.SortByImpl is imported.